indexer/services/roundtable/src/tasks/update-affiliate-info.ts

```diff
@@ -0,0 +1,50 @@
+import { logger, stats } from '@dydxprotocol-indexer/base';
+import {
+  PersistentCacheTable, AffiliateInfoTable, PersistentCacheKeys, PersistentCacheFromDatabase,
+} from '@dydxprotocol-indexer/postgres';
+import { DateTime } from 'luxon';
+
+import config from '../config';
+
+const defaultLastUpdateTime: string = '2023-10-26T00:00:00Z';
+
+/**
+ * Update the affiliate info for all affiliate addresses.
+ */
+export default async function runTask(): Promise<void> {
```

Comment:
don't you have to update index.ts so this task is run?

Reply:
fixed